test/IDE/complete_override.swift

```diff
@@ -115,9 +115,6 @@ protocol ProtocolA {
   func protoAFunc()
   @objc optional func protoAFuncOptional()
 
-  @noreturn
-  func protoAFuncWithAttr()
-
```

Comment:
Thanks Ben. This change is coming up in the next patch. I need to land the preliminary stuff first so I can update the other repos.